pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix

```diff
@@ -62,7 +62,7 @@ in rec {
   ];
 
   mkYarnModules = {
-    name, # safe name and version, e.g. testcompany-one-modules-1.0.0
+    name ? "${pname}-${version}", # safe name and version, e.g. testcompany-one-modules-1.0.0
```

Comment:
maybe that can be a separate commit. I think it can be merged quite quickly

Reply:
It is :)

Reply:
Sorry, I meant separate PR. Since this PR is almost done, I don't have a problem with leaving as is.